### PR TITLE
e2e: Ensure that Bundles that contain CRDs and instances of those CRDs fail

### LINKED ISSUE #185: Introduce an e2e test that ensure that Bundles that contain CRDs and instances of those CRDs reports a failed installation state
Goal: introduce an e2e test that ensures that Bundles that contain CRDs and instance those CRDs eventually report a failed state.

### PR DESCRIPTION
**Pull request by timflannagan**
Introduce a new BI e2e test that's responsible for stamping out a Bundle that contains CRD definitions and instances of those CRDs and ensuring that the install eventually reports a failed status.

Closes #185
Related to #131 

Signed-off-by: timflannagan <timflannagan@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -145,13 +145,15 @@ kind-load-bundles: ## Load the e2e testdata container images into a kind cluster
 	$(CONTAINER_RUNTIME) build $(TESTDATA_DIR)/bundles/plain-v0/empty -t testdata/bundles/plain-v0:empty
 	$(CONTAINER_RUNTIME) build $(TESTDATA_DIR)/bundles/plain-v0/no-manifests -t testdata/bundles/plain-v0:no-manifests
 	$(CONTAINER_RUNTIME) build $(TESTDATA_DIR)/bundles/plain-v0/invalid-missing-crds -t testdata/bundles/plain-v0:invalid-missing-crds
+	$(CONTAINER_RUNTIME) build $(TESTDATA_DIR)/bundles/plain-v0/invalid-missing-crds -t testdata/bundles/plain-v0:invalid-crds-and-crs
```
Comment (akihikokuroda) [inline Makefile:148]: Should it be `$(TESTDATA_DIR)/bundles/plain-v0/invalid-crds-and-crs`?

File: testdata/bundles/plain-v0/invalid-crds-and-crs/manifests/resources.yaml [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+                    protocol:
+                      type: string
+                    serviceName:
+                      type: string
+                    serviceNamespace:
+                      type: string
+      served: true
+      storage: true
+      subresources:
+        status: {}
```
Comment (akihikokuroda) [inline testdata/bundles/plain-v0/invalid-crds-and-crs/manifests/resources.yaml:273]: Is Docker file in missing?

File: Makefile (hunk 1 above)
Comment (timflannagan) [reply Makefile:148]: Nice catch - I'd guess I messed this up during a rebase.

**Review by akihikokuroda — APPROVED** [review]
Looks good!  Thanks!

**Pull request merged**